src/Tizen.Peripheral/Tizen.Peripheral/Adc.cs

```diff
@@ -0,0 +1,86 @@
+﻿/*
+* Copyright (c) 2020 Samsung Electronics Co., Ltd All Rights Reserved
+*
+* Licensed under the Apache License, Version 2.0 (the License);
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an AS IS BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using NativeAdc = Interop.Peripheral.Adc;
+
+namespace Tizen.Peripheral
+{
+    /// <summary>
+    /// The class allows applications to use the platform ADC peripheral.
+    /// </summary>
+    /// <privilege>http://tizen.org/privilege/peripheralio</privilege>
+    public class Adc : IDisposable
+    {
+        /// <summary>
+        /// Native handle to ADC.
+        /// </summary>
+        protected IntPtr _handle;
```

Comment:
Why is this `_handle` `protected`? Do not open IntPtr type handle to public APIs.
```suggestion
        private IntPtr _handle;
```